boot/initramfs20dirs.go

```diff
@@ -50,6 +54,7 @@ var (
 func setInitramfsDirVars(rootdir string) {
 	InitramfsRunMntDir = filepath.Join(rootdir, "run/mnt")
 	InitramfsUbuntuDataDir = filepath.Join(InitramfsRunMntDir, "ubuntu-data")
+	InitramfsHostDataDir = filepath.Join(InitramfsRunMntDir, "host", "data")
```

Comment:
this should be ubuntu-data, as per spec we agreed to rbind mount /run/mnt/host to /host, and the final "host's ubuntu-data" should be called "ubuntu-data" no? Or did we change the name there?

Reply:
Updated